CONTRIBUTING.md

```diff
@@ -141,15 +141,9 @@ Be sure to re-run build & refresh to update after making changes!
 
 ## Optional: Improve Ergonomics on Mac and Linux
 
-### Enable `./make.py`
+### `./make.py` shorthand
 
-On Mac & Linux, you can enable running `make.py` using `./make.py` instead
-of `python make.py` as follows:
-
-1. Make sure you are in the root directory of the repo
-2. Run `chmod u+x make.py`
-
-You can run the make script with `./make.py` instead of `python make.py`.
+On Mac & Linux, you can run the make script as `./make.py` instead of `python make.py`.
```

Comment:
I think it's worth including a warning about `--install-completion` applying to all `make.py` instances rather than just the local one.